WooCommerce/WooCommerceTests/Extensions/DoubleWooTests.swift

```diff
@@ -23,8 +23,8 @@ class DoubleWooTests: XCTestCase {
         XCTAssertEqual(Double(999).humanReadableString(), "999")
         XCTAssertEqual(Double(999.99).humanReadableString(), "999")
         XCTAssertEqual(Double(999.99999).humanReadableString(), "999")
-        XCTAssertEqual(Double(1000).humanReadableString(), expectedLocalizedAbbreviation(for: 1000)) // "1.0k"
-        XCTAssertEqual(Double(1000.00001).humanReadableString(), expectedLocalizedAbbreviation(for: 1000)) // "1.0k"
+        XCTAssertEqual(Double(1000).humanReadableString(), "1.0k") // "1.0k"
+        XCTAssertEqual(Double(1000.00001).humanReadableString(), "1.0k") // "1.0k"
```

Comment:
All the changes in DoubleWooTests remove the private local function `expectedLocalizedAbbreviation(for:)`. I removed it because calculating an `expected result` by using the same method that you're supposed to be testing as the `actual result` means the tests will pass without testing the right thing. All expected results have returned to their string values with no calculations or string manipulations applied.

If this causes problems and the tests fail in your locale, please let me know. That means the `humanReadableString()` method is not correct.

Reply:
I confirm that all the Human readable formatter tests fail for me.

<img width="964" alt="Schermata 2019-12-09 alle 12 42 52" src="https://user-images.githubusercontent.com/495617/70433248-7ab4e400-1a81-11ea-8f59-63073d07c434.png">

Reply:
Unit tests updated. Try now?